doc/src/content/xdocs/spec.xml

```diff
@@ -254,14 +254,16 @@
         <section>
           <title>Fixed</title>
           <p>Fixed uses the type name <code>"fixed"</code> and supports
-          two attributes:</p>
+          the following attributes:</p>
 	  <ul>
 	    <li><code>name</code>: a string naming this fixed (required).</li>
 	    <li><em>namespace</em>, a string that qualifies the name;</li>
```

Comment:
A few questions not directly related to this PR: 
1) Should we add `(optional)` for namespace too ?
2) Why it uses `<em>` instead of `<code>` like all other attributes ?

Reply:
change <em> to <code> for namespace - at least I cant think of a reason why it's different?
i agree namespace should be marked optional (it is. it is also ignored is name has dots in it), but will leave that to a committer to decide

Reply:
You have changed two occurrences of `<em>` but there are two more. IMO, if we do this change then we should do it for all of the occurrences.